Add backend error linking to replays (#8389)
getsentry/sentry-docs: src/docs/product/session-replay/getting-started.mdx

- [Remix](/platforms/javascript/guides/remix/session-replay/)
- [Svelte](/platforms/javascript/guides/svelte/session-replay/)
- [Vue](/platforms/javascript/guides/vue/session-replay/)

> ## Backend Error Linking

Reviewer: I'm not about about calling Backend Error Linking because basically this is Tracing we're talking about.

If folks have Tracing setup, it should just work.

Author: Thoughts on changing it to backend error tracing to make it clearer?

Reviewer: How about calling it what the user will see: "Replay in backend errors"

And in the description adjust slightly to mention that for as long as distributed tracing is enabled, backedn errors and transactions will display a replay of what was going on on the frontend?

Author: I like that suggestion! Just updated the wording

> Backend error linking works via integrating replays with our tracing data model. When you have Sentry wired up on your frontend and backend, we have data connectivity end-to-end on your APIs and can link replays to backend endpoints.

Reviewer 2: ```suggestion
Backend error linking works by integrating replays with Sentry's tracing data model. When you have Sentry set up up for both your frontend and backend, there's end-to-end data connectivity on your APIs which can link replays to backend endpoints.
```

> With backend error linking, the **Replay Details** view includes backend errors in the Timeline, Breadcrumbs, Errors tab, and more. You'll also see a Replays tab and a preview of a replay in the **Issue Details** page for any backend error that has a linked replay.

Reviewer 2: ```suggestion
If backend error linking is enabled, you'll be able to view backend errors in the Timeline, Breadcrumbs, Errors tab, and other places when you go to the **Replay Details** view. You'll also see a Replays tab and replay previews in the **Issue Details** page for any backend error that has a linked replay.
```

For backend error linking to work, make sure you've set up trace propagation in your backend projects. For example:

```
Sentry.init({
  tracePropagationTargets: ["third-party-site.com", /^https:\/\/yourserver\.io\/api/],
});
```

> Currently, backend error linking is supported for the following SDKs:

Reviewer: Worth indicating that's the version we introduced support, any version after that included, and latest is always advised:

```suggestion
Currently, backend error linking is supported for the following SDK versions or newer:
```
